airbyte-config/models/src/main/resources/types/StandardWorkspace.yaml

```diff
@@ -39,3 +39,6 @@ properties:
       if not set or false, the configuration is active. if true, then this
       configuration is permanently off.
     type: boolean
+  failureNotificationsWebhook:
```

Comment:
nit:
```suggestion
  failureNotificationWebhook:
```

Reply:
i know we are making a lot of tradeoffs for shipping this quickly, but i'm not sure i like this short cut. we 100% know that we are going to add more types of notifications. I don't think we want to add each one as a new field on the workspace, right?

should we be doing a list of notification configurations? and a notification configuration is a one of the different notification types? for now the only time we would have is slack. i'm not married to the specifics of what i just mentioned, but we should be building this so that the next notification type we add doesn't require us to throwaway this whole PR.

Reply:
100% agreeing with that. The PR is too specific.